v20.2/disaster-recovery.md

```diff
@@ -0,0 +1,365 @@
+      <td>When the regions come back online, try <a href="cockroach-start.html">restarting the nodes</a> in the cluster. <br><br>If the regions do not come back online and nodes are lost or destroyed, try <a href="restore.html">restoring the latest cluster backup</a> into a new cluster.<br><br>You can also <a href="https://support.cockroachlabs.com">contact Cockroach Labs support</a> for assistance.</td>
+    </tr>
+  </tbody>
+</table>
+
+<span style="color:#228B22"><b>√</b></span> = Available
+<span style="color:#FF0000"><b>X</b></span> = Outage
+
+{{site.data.alerts.callout_info}}
+When using Kubernetes, recovery actions happen automatically in many cases and no action needs to be taken.
+{{site.data.alerts.end}}
+
+## Data failure
+
+When dealing with data failure due to bad actors, rogue applications, or data corruption, domain expertise is required to identify the affected rows and determine how to to remedy the situation (e.g., remove the incorrectly inserted rows, insert deleted rows, etc.). However, there are a few actions that you can take for short-term remediation:
+
+- If you are within the garbage collection window, [run differentials](#run-differentials).
+- If you have a backup file, [restore to a point in time](#restore-to-a-point-in-time).
+- If your cluster is running and you do not have a backup with the data you need, [create a new backup](#create-a-new-backup).
+- To [recover from corrupted data in a database or table](#recover-from-corrupted-data-in-a-database-or-table), restore the corrupted object.
+
+{{site.data.alerts.callout_success}}
+To give yourself more time to recover and clean up the corrupted data, put your application in “read only” mode and only run [`AS OF SYSTEM TIME`](as-of-system-time.html) queries from the application.
+{{site.data.alerts.end}}
+
+### Run differentials
+
+If you are within the [garbage collection window](configure-replication-zones.html#replication-zone-variables) (default is 25 hours), run [`AS OF SYSTEM TIME`](as-of-system-time.html) queries and use [`CREATE TABLE AS … SELECT * FROM`](create-table-as.html) to create comparison data and run differentials to find the offending rows to fix.
+
+If you are outside of the garbage collection window, you will need to use a [backup](backup.html) to run comparisons.
+
+### Restore to a point in time
+
+- If you are a core user, use a [backup](backup.html) that was taken with [`AS OF SYSTEM TIME`](as-of-system-time.html) to restore to a specific point.
+- If you are an enterprise user, use your [backup](backup.html) file to [restore to a point in time](backup-and-restore-advanced-options.html#point-in-time-restore) where you are certain there was no corruption.
+
+### Create a new backup
+
+If your cluster is running and you do not have a backup that encapsulates the time you want to [restore](restore.html) to, there are two actions you can take:
```

Comment:
Maybe a clarification that the data you want to recover still has to be in the GC window?